ghutil/ghutil_test.go

```diff
@@ -442,3 +442,103 @@ func TestCheckPullRequestCompliance_OneCompliantOneNot(t *testing.T) {
 	assert.Equal(t, "Committer of one or more commits is not listed as a CLA signer, either individual or as a member of an organization.", nonComplianceReason)
 	assert.Nil(t, err)
 }
+
+func TestCheckProcessPullRequest_Compliant_Approve(t *testing.T) {
+	setUp(t)
+	defer tearDown(t)
+
+	expectRepoLabels(orgName, repoName, true, true, false)
+	repoClaLabelStatus := ghc.GetRepoClaLabelStatus(ctx, orgName, repoName)
+
+	claSigners := config.ClaSigners{
+		People: []config.Account{
+			{
+				Name:  "John Doe",
+				Email: "john@example.com",
+				Login: "john-doe",
+			},
+		},
+	}
+	var prNum = 42
+	var num *int
+	num = &prNum
+	var prTitle = "Fix Things"
+	var title *string
+	title = &prTitle
+	pull := &github.PullRequest{Number: num, Title: title}
+
+	var commits []*github.RepositoryCommit
+	mockGhc.PullRequests.EXPECT().ListCommits(ctx, orgName, repoName, pullNumber, nil).Return(commits, nil, nil)
+	mockGhc.Issues.EXPECT().ListLabelsByIssue(ctx, orgName, repoName, prNum, nil).Return(nil, nil, nil)
+	mockGhc.Issues.EXPECT().AddLabelsToIssue(ctx, orgName, repoName, prNum, []string{"cla: yes"}).Return(nil, nil, nil)
+	var reviewBody = ""
+	var body *string
+	body = &reviewBody
```

Comment:
Just inline `&""` below where you use `body`? Or if that's not possible, just inline `&reviewBody`? Similarly below and in the other test method.